templates/firefox-ios.xliff

```diff
@@ -539,6 +539,10 @@
         <source>Jump Back In</source>
         <note>Title for the Jump Back In section. This section allows users to jump back in to a recently viewed tab</note>
       </trans-unit>
+      <trans-unit id="ActivityStream.JumpBackIn.TabGroup.SiteCount" xml:space="preserve">
+        <source>%d sites</source>
+        <note>On the Firefox homepage in the Jump Back In section, if a Tab group item - a collection of grouped tabs from a related search - exists, underneath the search term for the tab group, there will be a subtitle with how many tabs (or sites) are in that group. The placeholder is for a number. It will read '5 sites' or similar.</note>
```

Comment:
Seems like I didn't see this before. Unfortunately we don't handle properly plural forms (and most languages don't handle plural like English). To workaround this, please update string to "sites: %d"